OxfordBodleian/Juel-Jensen/BDLaethf29.xml

```diff
@@ -348,14 +348,13 @@ type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?>
                             <supportDesc>
                                 <support>
                                     <material key="parchment"/>
-                                    <p>A hole on <locus target="#54"/>, presumably from parchment
-                                        making.</p>
-                                    <p>A tear carefully amended on <locus target="#42"/>.</p>
+                                    <p>A hole on <locus target="#54"/> from parchment
+                                        making. A tear carefully amended on <locus target="#42"/>.</p>
                                 </support>
                                 <extent>
-                                    <measure unit="page">123</measure>
-                                    <measure unit="leaf" type="blank">5</measure>
-                                    <!--page-->
+                                    <measure unit="leaf" quantity="123">123</measure>
+                                    <measure unit="quire" quantity="14">14</measure>
```

Comment:
I added a comment. I agree with your analysis, when you correct the quire structure, you could also correct this statement.

Reply:
I think we should take into consideration that the sewing in that point is broken and it could be that fold. 1-3 were sewn together originally...fol. 3 seems to have been folded along the inner margin to create a stub. Perhaps it was wrapped around fols. 1-2. But this is just speculation.